metricflow/test/model/validations/test_reserved_keywords.py

```diff
@@ -19,18 +19,6 @@ def test_no_reserved_keywords(simple_model__with_primary_transforms: UserConfigu
     assert len(issues) == 0
 
 
-def test_reserved_keywords_in_sql_table(simple_model__with_primary_transforms: UserConfiguredModel) -> None:  # noqa: D
```

Comment:
We might want to keep this validation for the string output of the NodeRelation or whatever we wind up calling it, but I think deleting this test is ok for now.

Reply:
Brought this back and modified to validate NodeRelation